### PR TITLE
Add alert rules for PD_cluster_slow_tikv_nums

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <hustmh@gmail.com>

### What is changed, added or deleted? (Required)

Add alert rules for PD_cluster_slow_tikv_nums

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: alert-rules.md [hunk 1]
```diff
@@ -399,6 +399,19 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
...
     * 根据 label 配置（如果有这个配置的话）来检查是否有可以补副本的 store。
 
+#### `PD_cluster_slow_tikv_nums`
+
+* 报警规则：
+    `(sum(pd_cluster_status{type="store_slow_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
+
+* 规则描述：
+
+    某一个 TiKV 被检测为慢节点。慢节点的检测由 TiKV `raftstore.inspect-interval` 参数控制，参见 [TiKV 配置文件描述](/tikv-configuration-file#inspect-interval)。
```
Comment (Oreoxmt) [inline alert-rules.md:409]:
```suggestion
    某一个 TiKV 被检测为慢节点。慢节点的检测由 TiKV `raftstore.inspect-interval` 参数控制，参见 [TiKV 配置文件描述](/tikv-configuration-file.md#inspect-interval)。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@nolouch PTAL

**Review by nolouch — APPROVED** [review]

**Pull request merged**